README.md

```diff
@@ -109,10 +123,11 @@ STATICFILES_DIRS = (
 Restart the apache server to put all changes into effect. 
 
 ```
-/etc/init.d/apache2 reload
+sudo /etc/init.d/apache2 reload
 ```
 
-###Migrate the Database Schema using South
```

Comment:
or I can add a bit of commentary text above it saying that it should be run if there are untracked migrations